BUILDING

```diff
@@ -1 +1,13 @@
-To build Ion, use the build.sh or build.py scripts in the ion/ directory.
+1. Install dependencies
+  - gyp
+  - clang
+  - libc++-dev
+  - libc++abi-dev
+
```

Comment:
That is a bit vague. Which version of Clang is required?

Sorry to dump this on a closed pull request. I would have raised a ticket, but you don't seem to have issues enabled on this github repo.